tests/trainer/test_dataloaders.py

```diff
@@ -36,63 +42,13 @@ class CurrentTestModel(
     hparams = tutils.get_default_hparams()
     model = CurrentTestModel(hparams)
 
-    # percent check < 0
-
-    # logger file to get meta
-    trainer_options = dict(
-        default_root_dir=tmpdir,
-        max_epochs=1,
-        train_percent_check=-0.1,
-    )
-
-    # fit model
-    trainer = Trainer(**trainer_options)
-
-    with pytest.raises(ValueError):
-        trainer.fit(model)
-
```

Comment:
just double checking, was it intentional to remove these tests? was it duplicated?

Reply:
No test shall be removed unless it was merged with using pytest parameters...

Reply:
No test shall be removed unless it was merged with using pytest parameters...

Reply:
oh yes, missed that, it is parametrized, very nice!